packages/ic-certification/Cargo.toml

```diff
@@ -32,4 +32,5 @@ serde_cbor.workspace = true
 rstest.workspace = true
 
 [features]
-default = ['serde', 'serde_bytes']
```

Comment:
It didn't make sense to have two separate features before, so it's combined into one now.